onnxruntime/core/framework/graph_partitioner.cc

```diff
@@ -193,16 +193,18 @@ Status GraphPartitioner::Partition(Graph& graph, bool export_dll, FuncManager& f
   std::vector<Node*> nodes_need_inline;
   for (auto& node : graph.Nodes()) {
     if (node.GetExecutionProviderType().empty()) {
+      graph.InitFunctionBodyForNode(node);
       auto node_func = node.GetFunctionBody();
       if (nullptr == node_func) {
```

Comment:
Could we do a delayed init inside of node.GetFunctionBody() instead of having to call Graph::InitFunctionBodyForNode?

Node has an internal member for the graph that contains it so it looks like that should be possible.

Reply:
I did not realize Node has graph_ as an internal member. I will move this call inside GetFunctionBody